### PR TITLE
Feature/examples code snippet

### PR DESCRIPTION
**Pull request by TomVanDenBroeck**
## PR Checklist

This PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] A [changelog entry](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/guidelines/CHANGELOG.md) was added

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [*] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
added code-snippet example

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/code-snippet/README.md [hunk 1]
```diff
@@ -24,14 +25,31 @@ You can also add additional languages if that's needed:
 
 An overview of all CDN links can be found [here](https://cdnjs.com/libraries/highlight.js/)
 
-## Installation
-```
-npm install @acpaas-ui/code-snippet --save
+## Usage
+
+```javascript
```
Comment (TriangleJuice) [inline packages/code-snippet/README.md]: `typescript`

File: packages/code-snippet/README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
-# @acpaas-ui/code-snippet
-*A component for displaying a code snippet formatted by highlight.js*
+# @acpaas-ui/ngx-components/code-snippet
+
+This module contains a component for displaying a code snippet formatted by highlight.js
```
Comment (TriangleJuice) [inline packages/code-snippet/README.md]: I should only list the dependencies. I would delete all the other text.

File: packages/code-snippet/README.md [hunk 3]
```diff
@@ -24,14 +25,31 @@ You can also add additional languages if that's needed:
...
+| `@Input() processMarkdown: boolean;` | `false` | When having some markdown that contains code snippets. |
+| `@Input() scrollable: boolean;` | `true` | Boolean for when code snippet should not have a horizontal scrollbar when the code snippet is not wide enough. |
+
+#### Example
 
-``` typescript
+##### A single code snippet
+
+```
 import { CodeSnippetModule } from '@acpaas-ui/code-snippet';
```
Comment (TriangleJuice) [inline packages/code-snippet/README.md]: Wut? 🙃

File: packages/code-snippet/examples/code-snippet/pages/demo/demo.page.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<h3 class="u-margin-bottom">Code snippet</h3>
```
Comment (TriangleJuice) [inline packages/code-snippet/examples/code-snippet/pages/demo/demo.page.html]: Ik zie nergens de complete code staan. T.t.z. ik zie geen controller code, enkel html.

File: packages/code-snippet/README.md [hunk 5]
```diff
@@ -8,30 +9,31 @@ This package is build depending on:
 - and the [corresponding angular2 npm package](https://www.npmjs.com/package/angular2-highlight-js).
 - the [Marker npm package](https://www.npmjs.com/package/marked)
 
-*If you were to link this component locally, perform an `npm install` for the above npm packages in your app first.*
+## Usage
 
-### Highlight.js CDN's
-Add the following CDN's to your project:
+```typescript
+import 'CodeSnippetModule' from '@acpaas-ui/ngx-components/code-snippet'`;
```
Comment (lievenvg) [inline packages/code-snippet/README.md]: import { CodeSnippetModule }

File: packages/code-snippet/README.md [hunk 6]
```diff
@@ -8,30 +9,31 @@ This package is build depending on:
...
+| `@Input() processMarkdown: boolean;` | `false` | When having some markdown that contains code snippets. |
+| `@Input() scrollable: boolean;` | `true` | Boolean for when code snippet should not have a horizontal scrollbar when the code snippet is not wide enough. |
+
+#### Example
 
-Import the CodeSnippetModule in **app.module.ts**
+##### A single code snippet
 
-``` typescript
+```
```
Comment (lievenvg) [inline packages/code-snippet/README.md]: For consistency with the other examples, better to provide syntax typing.
`typescript`

File: packages/code-snippet/README.md [hunk 7]
```diff
@@ -42,13 +44,7 @@ import { CodeSnippetModule } from '@acpaas-ui/code-snippet';
...
 export class AppModule {}
 ```
-
-## Usage
-
-### A single code snippet
-In *your.component.ts* file:
-
-``` typescript
+```
```
Comment (lievenvg) [inline packages/code-snippet/README.md]: Here also. `typescript`
And watch your indenting.

File: packages/code-snippet/README.md [hunk 8]
```diff
@@ -69,42 +65,33 @@ public codeExampleJS = `
     }
     greetMe('World');`;
 ```
-
-In *your.component.html* file:
-
-``` html
+```
 <aui-code-snippet [codeSnippet]="codeExampleJSON"></aui-code-snippet>
```
Comment (lievenvg) [inline packages/code-snippet/README.md]: `html`

File: packages/code-snippet/README.md [hunk 9]
```diff
@@ -141,12 +122,6 @@ In *your.component.html* file:
...
-
-#### processMarkdown
-
-`boolean`: `false` by default but if you need to include some Markdown containing a code snippet, you can set it to `true` and just add the content of the markdown file to `[codeSnippet]`.
-
-#### scrollable
+## Contributing
 
-`boolean`: `true` by default. Change to option to false if the code-snippet should not have a horizontal scrollbar when the code-snippet is not wide enough.
+Visit our [Contribution Guidelines](./contribute.md) for more information on how to contribute.
```
Comment (lievenvg) [inline packages/code-snippet/README.md]: Update your contribution link

File: packages/code-snippet/examples/code-snippet/pages/demo/demo.page.ts [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+    }
+];`;
+
+	public html3 = `public codeExampleJS =
+function greetMe(yourName) {
+    alert('Hello ' + yourName);
+}
+greetMe('World');`;
+
+
```
Comment (lievenvg) [inline packages/code-snippet/examples/code-snippet/pages/demo/demo.page.ts]: Remove Enter

File: packages/code-snippet/README.md [hunk 11]
```diff
@@ -8,30 +9,31 @@ This package is build depending on:
...
 
-- `<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.10.0/languages/typescript.min.js"></script>`
-- `<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.10.0/languages/scss.min.js"></script>`
-- ...
+#### API
 
-An overview of all CDN links can be found [here](https://cdnjs.com/libraries/highlight.js/)
+| Name         | Description | Documentations |
+| -----------  | ------ | -------------------------- |
+| `@Input() codeSnippet: string;` | `''` | Add your code snippet here. |
```
Comment (TriangleJuice) [inline packages/code-snippet/README.md]: The initial value of `codeSnippet` is not defined, so it should say `-`.

File: packages/code-snippet/README.md [hunk 12]
```diff
@@ -69,42 +65,33 @@ public codeExampleJS = `
...
-``` typescript
+```typescript
 public codeExamples = [this.codeExampleJS, this.codeExampleJSON];
 ```
-
-In *your.component.html* file:
-
-``` html
+```html
 <aui-code-snippet
```
Comment (TriangleJuice) [inline packages/code-snippet/README.md]: I don't see this example in your example app.

File: packages/code-snippet/examples/code-snippet/pages/demo/demo.page.html [hunk 13]
```diff
@@ -0,0 +1,49 @@
+<h1 class="h3 u-margin-bottom">Code snippet</h1>
+<div class="u-margin-bottom">
+    <aui-code-snippet [codeSnippet]="codeExampleJS1"></aui-code-snippet>
+</div>
+
+<div class="u-margin-bottom">
+    <aui-code-snippet [codeSnippet]="codeExampleHTML1"></aui-code-snippet>
+</div>
+
+<h2 class="h4 u-margin-bottom">Code snippet Example: single text</h2>
```
Comment (TriangleJuice) [inline packages/code-snippet/examples/code-snippet/pages/demo/demo.page.html]: Kan je hier telkens het stukje `Code snippet Example: ` weghalen, a.u.b., zodat de title gewoon `Single text` wordt.

**Pull request merged**